hyperparam.ini

```diff
@@ -37,7 +37,7 @@
     batchTC_A = 1
     batchTC_G = 1
     batchTC_B = 6
-    no_mss = False
+    no_mss = False #used for factor too
```

Comment:
Should this be a separate argument with the new cleaned up code?

Reply:
yes it should be one for both. But in any case CLI should not contain arguments that people will not change. I.e. i'll run both chose the best then remove the hyperparameter from CLI (although it's perfectly fine in the function)